....F
======================================================================
FAIL: test_solve_2 (__main__.TestDiplomacy)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/Users/mariojuguilon/Documents/cs-330e/diplomacy/TestDiplomacy.py", line 47, in test_solve_2
    self.assertEqual("A [DEAD]\n\
AssertionError: 'A [DEAD]\n                          B Austin\n     [28 chars]on\n' != 'A [DEAD]\nB Austin\nC [DEAD]\n'
  A [DEAD]
-                           B Austin
-                           C Houston
+ B Austin
+ C [DEAD]


----------------------------------------------------------------------
Ran 5 tests in 0.002s

FAILED (failures=1)
